osdep/timer.c

```diff
@@ -94,7 +94,7 @@ struct timespec mp_time_ns_to_realtime(int64_t time_ns)
     struct timespec ts = {0};
 
 #if !HAVE_WIN32_INTERNAL_PTHREADS
-    if (get_realtime(&ts) != 0)
+    if (get_realtime(&ts) == 0)
         return ts;
```

Comment:
this is most certainly incorrect as it means the `time_ns` argument is entirely unused

Reply:
Hmm, you're right. I didn't realize that because this appeared to fix the issue